### PR TITLE
Explain the name, two small typos

### PR DESCRIPTION
**Pull request by manuelottlik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/cloudevents/badge)](https://clomonitor.io/projects/cncf/cloudevents)
 [![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/7295/badge)](https://bestpractices.coreinfrastructure.org/projects/7295)
 
-The xRegistry project defines an abstract model for how to manage metadata
+The xRegistry project (short for "extensible registry") defines an abstract model for how to manage metadata
```
Comment (duglin) [inline README.md:9]: Minor thing, but can you tweak this so that things wrap at 80 columns?

**Comment by duglin** [comment]
LGTM

**Review by jskeet — APPROVED** [review]
LGTM

**Comment by deissnerk** [comment]
LGTM

**Comment by duglin** [comment]
Approved on the 6/13 call.

**Pull request merged**